source/_integrations/rainmachine.markdown

```diff
@@ -49,7 +49,7 @@ Services accept either device IDs or entity IDs, depending on the nature of the
 
 ### `rainmachine.pause_watering`
 
-Pause all watering activities for a number of seconds.
+Pause all watering activities for a number of seconds. After the pause is complete, the previous watering activities will resume.
```

Comment:
Can the 12 hour restriction be mentioned either here, or in the table, to head off the user stumbling upon it later and having to redo something?